src/pages/countries.js

```diff
@@ -0,0 +1,82 @@
+import { useQuery, gql } from "@apollo/client";
+import { Link, useLocation, useHistory } from "react-router-dom";
+import CountriesList from "../components/CountriesList";
+import { useState, useEffect } from "react";
+
+export const COUNTRIES_QUERY = gql`
+  query allCountries($limit: Int, $skip: Int) {
+    countries(limit: $limit, skip: $skip) {
+      id
+      name
+      vatRate
+      languages {
+        id
+        name
+      }
+    }
+  }
+`;
```

Comment:
In real world we will probably keep queries encapsulated in files but this is totally ok in that scenarion 👍

Reply:
I encapsulated the queries into different files in incoming commit